prowler/lib/mutelist/mutelist.py

```diff
@@ -453,7 +454,10 @@ def validate_mutelist(mutelist: dict) -> dict:
             validate(mutelist, schema=mutelist_schema)
             return mutelist
         except Exception as error:
-            logger.error(
-                f"{error.__class__.__name__} -- Mutelist YAML is malformed - {error}[{error.__traceback__.tb_lineno}]"
-            )
+            if raise_on_exception:
+                raise error
+            else:
+                logger.error(
+                    f"{error.__class__.__name__} -- Mutelist YAML is malformed - {error}[{error.__traceback__.tb_lineno}]"
+                )
             return {}
```

Comment:
```suggestion
                return {}
```

I think this miss one level indentation, right?

Reply:
Functionally it's the exact same behavior either way, as long as `raise_on_exception` is false, it will always return `{}` in the case that there is an exception. I prefer it this way just to make that clear that we want to return an empty dict in that case.